src/PawsyModules/FilterMatcher/FilterMatcher.cs

```diff
@@ -74,12 +76,91 @@ public override SlashCommandBundle OnCommandsDeclared(SlashCommandBuilder builde
+                )
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("regex")
+                    .WithType(ApplicationCommandOptionType.String)
+                    .WithDescription("The REGEX for the rule")
+                    .WithRequired(true)
+                )
             )
             .AddOption(
                 new SlashCommandOptionBuilder()
                 .WithType(ApplicationCommandOptionType.SubCommand)
                 .WithName("edit")
                 .WithDescription("Edit an existing filter")
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("rule-id")
+                    .WithType(ApplicationCommandOptionType.Integer)
+                    .WithDescription("The ID of the rule to edit")
+                    .WithMinValue(0)
+                    .WithRequired(true)
+                )
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("name")
+                    .WithType(ApplicationCommandOptionType.String)
+                    .WithDescription("The new name for the rule")
+                )
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("regex")
+                    .WithType(ApplicationCommandOptionType.String)
+                    .WithDescription("The new REGEX for the rule")                    
+                )
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("channel")
+                    .WithType(ApplicationCommandOptionType.Channel)
+                    .WithDescription("Add or remove a channel from the list of filtered channels")
+                )
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("delete")
+                    .WithType(ApplicationCommandOptionType.Boolean)
+                    .WithDescription("Should the message be deleted?")
+                )
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("reply")
+                    .WithType(ApplicationCommandOptionType.String)
+                    .WithDescription("What will pawsy reply to the user?")
+                )
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("warn-staff")
+                    .WithType(ApplicationCommandOptionType.Boolean)
+                    .WithDescription("Should pawsy send a message in the alert channel?")
+                )
+                .AddOption(
+                    new SlashCommandOptionBuilder()
+                    .WithName("warn-color")
+                    .WithType(ApplicationCommandOptionType.String)
+                    .WithDescription("The color of the message report embeded")
```

Comment:
I think the description should say this is in hex notation RRGGBB and show an example or something